### PR TITLE
Test/olas week tests

### PR DESCRIPTION
**Pull request by annasambrook**
## Proposed changes

Tests added to cover the olas week skill

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/valory/skills/olas_week_abci/tests/test_behaviours.py [hunk 1]
```diff
@@ -0,0 +1,972 @@
...
+        "conversation_id": "3",
+    },
+    {
+        "author_id": "1286010187325812737",
+        "text": "tweet_text_4",
+        "id": "4",
+        "created_at": "2023-06-01T12:15:00Z",
+        "conversation_id": "4",
+    },
+]
```
Comment (Adamantios) [inline packages/valory/skills/olas_week_abci/tests/test_behaviours.py:242]: Why not fixtures?

File: packages/valory/skills/olas_week_abci/behaviours.py [hunk 2]
```diff
@@ -467,16 +467,6 @@ def _get_tweets(
...
-        if number_of_tweets_remaining_today <= 0:
-            self.context.logger.info(
-                "Cannot retrieve tweets, max number of tweets reached for today"
-            )
-            return {
-                "tweets": None,
-                "error": ERROR_API_LIMITS,
-                "number_of_tweets_pulled_today": number_of_tweets_pulled_today,
-                "sleep_until": self.synchronized_data.sleep_until,
-            }
```
Comment (Adamantios) [inline packages/valory/skills/olas_week_abci/behaviours.py:479]: Why is this removed?

**Review by Adamantios — APPROVED** [review]

File: packages/valory/skills/olas_week_abci/behaviours.py (hunk 2 above)
Comment (dvilelaf) [reply packages/valory/skills/olas_week_abci/behaviours.py:479]: Anna realised we have a code duplucation and we were checking for the tweet limits twice.
